File: app/src/main/java/com/oshi/genericsettings/java/utils/ArrayUtils.java
package com.oshi.genericsettings.java.utils;


import com.oshi.libgenericsettings.data.CheckableSubItem;

import java.util.Arrays;
import java.util.List;

public class ArrayUtils {

    public static List<String> getNewSimpleItemsList() {
        return Arrays.asList("First", "Rocket man", "Third!", "Last");
    }

    public static List<CheckableSubItem> getNewCheckableItemsList() {
        return Arrays.asList(
                new CheckableSubItem("First"),
                new CheckableSubItem("Second"),
                new CheckableSubItem("Third"));
    }

    public static List<String> createSimpleItemsArray(String... items) {
        return Arrays.asList(items);
    }

}

File: app/src/main/java/com/oshi/genericsettings/kotlin/utils/ArrayUtils.kt
package com.oshi.genericsettings.kotlin.utils

import com.oshi.libgenericsettings.data.CheckableSubItem
import java.util.*
import kotlin.collections.ArrayList

/**
 * Created by udioshi on 1/2/2018.
 */
class ArrayUtils {

    companion object {
        fun getNewSimpleItemsList() : List<String> {
            return Arrays.asList("First", "Rocket man", "Third!", "Last")
        }

        fun getNewCheckableItemsList() : List<CheckableSubItem> {
            return Arrays.asList(
                    CheckableSubItem("First"),
                    CheckableSubItem("Second"),
                    CheckableSubItem("Third"))
        }

        fun createSimpleItemsArray(vararg items : String) : List<String> {
            val list = ArrayList<String>()
            list += items
            return list
        }
    }

}